packages/p2p/src/components/my-ads/toggle-ads.jsx

```diff
@@ -3,12 +3,14 @@ import PropTypes            from 'prop-types';
 import { ToggleSwitch }     from '@deriv/components';
 import classNames           from 'classnames';
 import { localize }         from 'Components/i18next';
+import { requestWS }        from 'Utils/websocket';
 import './my-ads.scss';
 
-const ToggleMessage = ({ is_enabled, className }) => {
+const ToggleMessage = ({ is_enabled, className, error }) => {
     return (
         <p className={className}>
-            {is_enabled ? localize('Your ads are running') : localize('Your ads are paused')}
+            {error && error}
+            {is_enabled && !error ? localize('Your ads are running') : localize('Your ads are paused')}
```

Comment:
the way this condition is worded, if there is error we will show your ads are paused. better make these two lines into one, if error show error else if is_enabled show running else paused